sdk/spring/azure-spring-boot/pom.xml

```diff
@@ -326,8 +327,9 @@
                 <include>com.microsoft.azure:azure-servicebus-jms:[0.0.7]</include> <!-- {x-include-update;com.microsoft.azure:azure-servicebus-jms;external_dependency} -->
                 <include>com.github.spotbugs:spotbugs-annotations:[4.2.0]</include> <!-- {x-include-update;com.github.spotbugs:spotbugs-annotations;external_dependency} -->
                 <include>com.fasterxml.jackson.core:jackson-databind:[2.12.2]</include> <!-- {x-include-update;com.fasterxml.jackson.core:jackson-databind;external_dependency} -->
-                <include>net.minidev:json-smart:[2.3]</include> <!-- {x-include-update;net.minidev:json-smart;external_dependency} -->
-                <include>com.microsoft.azure:msal4j:[1.9.1]</include> <!-- {x-include-update;com.microsoft.azure:msal4j;external_dependency} -->
+                <!-- TODO: add {x-include-update;net.minidev:json-smart;external_dependency} this tag below as part of PR: https://github.com/Azure/azure-sdk-for-java/pull/20770-->
```

Comment:
DIsabling this update.
As spring team aims to update this via this PR (#20770) post their releases.